book/src/development/common_tools_writing_lints.md

```diff
@@ -192,7 +192,7 @@ functions to deal with macros:
 - `span.from_expansion()`: detects if a span is from macro expansion or
   desugaring. Checking this is a common first step in a lint.
 
-   ```rust
+   ```rust,ignore
```

Comment:
The `ignore` flag is AFAIK only used by rustdoc, I know that there are tools to also run the code in md book, though. Is there a reason why you think it's better to add them? For me, it seems like this expects a certain type of markdown to be used and makes it less compatible for little reason :thinking: 

The rendering seems to work on GH just fine either way:
```rust,ignore
let x = 0;
```

Reply:
@xFrednet oops, just saw your reply, thx for patience :)  The reason I added it there was because some IDEs (and most like all IDEs in the future) treat any code inside the triple backticks as valid code, and try to help with it. In this case, the code is incorrect (e.g. it has a tripple dot instead of a real code: `fn main() { ... }`) -- which means IDEs show it as an error.